pkg/persistence/account/loader/load.go

```diff
@@ -207,6 +207,32 @@ func decode(in io.ReadCloser) (map[string]any, error) {
 }
 
 func transform(resources *persistence.Resources) *account.Resources {
```

Comment:
i think the benefit here is the possibility to return nil. whereas otherwise you would need to construct a account.Resources value prefferably with valid (non nil) fields.